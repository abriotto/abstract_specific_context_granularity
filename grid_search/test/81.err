  0%|                                                                                                                                                                                                    | 0/99 [00:00<?, ?it/s]  1%|#8                                                                                                                                                                                          | 1/99 [00:00<00:16,  6.03it/s]  3%|#####6                                                                                                                                                                                      | 3/99 [00:00<00:08, 11.08it/s]  5%|#########4                                                                                                                                                                                  | 5/99 [00:00<00:08, 10.59it/s]  7%|#############2                                                                                                                                                                              | 7/99 [00:00<00:08, 10.73it/s]  9%|#################                                                                                                                                                                           | 9/99 [00:00<00:09,  9.02it/s] 10%|##################8                                                                                                                                                                        | 10/99 [00:01<00:10,  8.77it/s] 11%|####################7                                                                                                                                                                      | 11/99 [00:01<00:10,  8.19it/s] 12%|######################6                                                                                                                                                                    | 12/99 [00:01<00:10,  8.30it/s] 13%|########################5                                                                                                                                                                  | 13/99 [00:01<00:10,  7.87it/s] 14%|##########################4                                                                                                                                                                | 14/99 [00:01<00:11,  7.62it/s] 15%|############################3                                                                                                                                                              | 15/99 [00:01<00:11,  7.59it/s] 16%|##############################2                                                                                                                                                            | 16/99 [00:01<00:11,  7.05it/s] 17%|################################1                                                                                                                                                          | 17/99 [00:02<00:12,  6.76it/s] 19%|###################################8                                                                                                                                                       | 19/99 [00:02<00:10,  7.47it/s] 23%|###########################################4                                                                                                                                               | 23/99 [00:02<00:06, 12.16it/s] 25%|###############################################2                                                                                                                                           | 25/99 [00:02<00:06, 11.03it/s] 27%|##################################################9                                                                                                                                        | 27/99 [00:03<00:08,  8.76it/s] 28%|####################################################8                                                                                                                                      | 28/99 [00:03<00:08,  8.09it/s] 29%|######################################################7                                                                                                                                    | 29/99 [00:03<00:09,  7.58it/s] 31%|##########################################################5                                                                                                                                | 31/99 [00:03<00:07,  8.97it/s] 32%|############################################################4                                                                                                                              | 32/99 [00:03<00:08,  8.28it/s] 33%|##############################################################3                                                                                                                            | 33/99 [00:03<00:08,  7.74it/s] 35%|##################################################################1                                                                                                                        | 35/99 [00:03<00:06,  9.81it/s] 37%|#####################################################################8                                                                                                                     | 37/99 [00:04<00:05, 10.64it/s] 39%|#########################################################################6                                                                                                                 | 39/99 [00:04<00:07,  8.55it/s] 40%|###########################################################################5                                                                                                               | 40/99 [00:04<00:07,  7.77it/s] 41%|#############################################################################4                                                                                                             | 41/99 [00:04<00:08,  7.24it/s] 43%|#################################################################################2                                                                                                         | 43/99 [00:05<00:07,  7.69it/s] 45%|#####################################################################################                                                                                                      | 45/99 [00:05<00:06,  7.96it/s] 47%|########################################################################################7                                                                                                  | 47/99 [00:05<00:06,  7.97it/s] 49%|############################################################################################5                                                                                              | 49/99 [00:05<00:05,  9.31it/s] 52%|################################################################################################3                                                                                          | 51/99 [00:05<00:04, 10.11it/s] 54%|####################################################################################################1                                                                                      | 53/99 [00:06<00:04,  9.64it/s] 56%|#######################################################################################################8                                                                                   | 55/99 [00:06<00:04,  9.69it/s] 58%|###########################################################################################################6                                                                               | 57/99 [00:06<00:03, 10.72it/s] 60%|###############################################################################################################4                                                                           | 59/99 [00:06<00:03, 10.19it/s] 62%|###################################################################################################################2                                                                       | 61/99 [00:06<00:03, 11.42it/s] 64%|#######################################################################################################################                                                                    | 63/99 [00:06<00:03, 10.61it/s] 66%|##########################################################################################################################7                                                                | 65/99 [00:07<00:03, 10.38it/s] 68%|##############################################################################################################################5                                                            | 67/99 [00:07<00:03,  9.96it/s] 70%|##################################################################################################################################3                                                        | 69/99 [00:07<00:03,  9.73it/s] 72%|######################################################################################################################################1                                                    | 71/99 [00:07<00:02, 10.23it/s] 74%|#########################################################################################################################################8                                                 | 73/99 [00:08<00:02,  8.92it/s] 76%|#############################################################################################################################################6                                             | 75/99 [00:08<00:02,  8.70it/s] 77%|###############################################################################################################################################5                                           | 76/99 [00:08<00:02,  8.14it/s] 78%|#################################################################################################################################################4                                         | 77/99 [00:08<00:02,  7.65it/s] 80%|#####################################################################################################################################################2                                     | 79/99 [00:08<00:02,  7.74it/s] 82%|#########################################################################################################################################################                                  | 81/99 [00:09<00:02,  7.95it/s] 84%|############################################################################################################################################################7                              | 83/99 [00:09<00:01,  9.17it/s] 87%|##################################################################################################################################################################4                        | 86/99 [00:09<00:01, 10.60it/s] 89%|######################################################################################################################################################################2                    | 88/99 [00:09<00:00, 11.97it/s] 91%|##########################################################################################################################################################################                 | 90/99 [00:09<00:00,  9.39it/s] 93%|#############################################################################################################################################################################7             | 92/99 [00:10<00:00,  8.06it/s] 95%|#################################################################################################################################################################################5         | 94/99 [00:10<00:00,  8.09it/s] 96%|###################################################################################################################################################################################4       | 95/99 [00:10<00:00,  7.73it/s] 98%|#######################################################################################################################################################################################2   | 97/99 [00:10<00:00,  9.42it/s]100%|###########################################################################################################################################################################################| 99/99 [00:11<00:00,  9.11it/s]100%|###########################################################################################################################################################################################| 99/99 [00:11<00:00,  8.97it/s]
  0%|                                                                                                                                                                                                    | 0/25 [00:00<?, ?it/s]  8%|###############                                                                                                                                                                             | 2/25 [00:00<00:01, 13.59it/s] 16%|##############################                                                                                                                                                              | 4/25 [00:00<00:02, 10.01it/s] 24%|#############################################1                                                                                                                                              | 6/25 [00:00<00:02,  9.03it/s] 32%|############################################################1                                                                                                                               | 8/25 [00:00<00:01, 11.27it/s] 40%|##########################################################################8                                                                                                                | 10/25 [00:00<00:01,  9.93it/s] 48%|#########################################################################################7                                                                                                 | 12/25 [00:01<00:01, 10.15it/s] 56%|########################################################################################################7                                                                                  | 14/25 [00:01<00:01,  9.22it/s] 64%|#######################################################################################################################6                                                                   | 16/25 [00:01<00:01,  8.87it/s] 72%|######################################################################################################################################6                                                    | 18/25 [00:01<00:00,  9.20it/s] 76%|##############################################################################################################################################1                                            | 19/25 [00:02<00:00,  8.49it/s] 80%|#####################################################################################################################################################6                                     | 20/25 [00:02<00:00,  7.75it/s] 84%|#############################################################################################################################################################                              | 21/25 [00:02<00:00,  7.38it/s] 92%|############################################################################################################################################################################               | 23/25 [00:02<00:00,  7.59it/s] 96%|###################################################################################################################################################################################5       | 24/25 [00:02<00:00,  7.32it/s]100%|###########################################################################################################################################################################################| 25/25 [00:02<00:00,  6.86it/s]100%|###########################################################################################################################################################################################| 25/25 [00:02<00:00,  8.49it/s]
