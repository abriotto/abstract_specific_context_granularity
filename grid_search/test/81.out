# ["--attributes=3", "--values=4", "--game_size=10", "--vocab_size_factor=3", "--n_epochs=100", "--batch_size=32", "--learning_rate=0.0005", "--hidden_size=256", "--temp_update=0.99", "--temperature=1"]
Creating train_ds and val_ds...

Creating test_ds...
{"loss": 0.675099790096283, "acc": 0.5627621412277222, "length": 2.3806416988372803, "mode": "train", "epoch": 1}
{"loss": 0.6103821396827698, "acc": 0.6828187108039856, "length": 2.4509150981903076, "mode": "test", "epoch": 1}
{"loss": 0.5455074906349182, "acc": 0.7297894358634949, "length": 2.7359635829925537, "mode": "train", "epoch": 2}
{"loss": 0.49455928802490234, "acc": 0.7752271294593811, "length": 2.935108184814453, "mode": "test", "epoch": 2}
{"loss": 0.46387746930122375, "acc": 0.7927107810974121, "length": 2.9941608905792236, "mode": "train", "epoch": 3}
{"loss": 0.4594157934188843, "acc": 0.7960054874420166, "length": 3.0116472244262695, "mode": "test", "epoch": 3}
{"loss": 0.4265271723270416, "acc": 0.8138819932937622, "length": 3.1292943954467773, "mode": "train", "epoch": 4}
{"loss": 0.4273220896720886, "acc": 0.814588189125061, "length": 3.2762062549591064, "mode": "test", "epoch": 4}
{"loss": 0.3930528163909912, "acc": 0.8330857157707214, "length": 3.1931536197662354, "mode": "train", "epoch": 5}
{"loss": 0.3839477598667145, "acc": 0.8380990028381348, "length": 3.311148166656494, "mode": "test", "epoch": 5}
{"loss": 0.3465195596218109, "acc": 0.8577054142951965, "length": 3.351414918899536, "mode": "train", "epoch": 6}
{"loss": 0.37677615880966187, "acc": 0.8309430480003357, "length": 3.424292802810669, "mode": "test", "epoch": 6}
{"loss": 0.3199000358581543, "acc": 0.870118260383606, "length": 3.5585274696350098, "mode": "train", "epoch": 7}
{"loss": 0.34430238604545593, "acc": 0.8629133105278015, "length": 3.8452579975128174, "mode": "test", "epoch": 7}
{"loss": 0.29369890689849854, "acc": 0.8810327649116516, "length": 3.7831356525421143, "mode": "train", "epoch": 8}
{"loss": 0.29903140664100647, "acc": 0.8827337026596069, "length": 3.737104892730713, "mode": "test", "epoch": 8}
{"loss": 0.27997079491615295, "acc": 0.8932299613952637, "length": 3.7907285690307617, "mode": "train", "epoch": 9}
{"loss": 0.29513898491859436, "acc": 0.8837940096855164, "length": 3.881863594055176, "mode": "test", "epoch": 9}
{"loss": 0.25889232754707336, "acc": 0.9008922576904297, "length": 3.9264535903930664, "mode": "train", "epoch": 10}
{"loss": 0.27091333270072937, "acc": 0.8981696963310242, "length": 4.0, "mode": "test", "epoch": 10}
{"loss": 0.24733255803585052, "acc": 0.9074591398239136, "length": 3.9834141731262207, "mode": "train", "epoch": 11}
{"loss": 0.280274361371994, "acc": 0.9029117822647095, "length": 4.0, "mode": "test", "epoch": 11}
{"loss": 0.23319776356220245, "acc": 0.9156141877174377, "length": 3.9955477714538574, "mode": "train", "epoch": 12}
{"loss": 0.25998562574386597, "acc": 0.8977537155151367, "length": 4.0, "mode": "test", "epoch": 12}
{"loss": 0.22776100039482117, "acc": 0.9163910150527954, "length": 3.9979608058929443, "mode": "train", "epoch": 13}
{"loss": 0.24831165373325348, "acc": 0.9128952026367188, "length": 4.0, "mode": "test", "epoch": 13}
{"loss": 0.21462605893611908, "acc": 0.9245135188102722, "length": 3.9985241889953613, "mode": "train", "epoch": 14}
{"loss": 0.2464504987001419, "acc": 0.9108984470367432, "length": 4.0, "mode": "test", "epoch": 14}
{"loss": 0.20682036876678467, "acc": 0.9251625537872314, "length": 3.998128652572632, "mode": "train", "epoch": 15}
{"loss": 0.23799507319927216, "acc": 0.9156404733657837, "length": 4.0, "mode": "test", "epoch": 15}
{"loss": 0.20110294222831726, "acc": 0.9299953579902649, "length": 3.99902606010437, "mode": "train", "epoch": 16}
{"loss": 0.2395644187927246, "acc": 0.913227915763855, "length": 4.0, "mode": "test", "epoch": 16}
{"loss": 0.20171889662742615, "acc": 0.9286068677902222, "length": 3.9984290599823, "mode": "train", "epoch": 17}
{"loss": 0.22421754896640778, "acc": 0.9178869128227234, "length": 4.0, "mode": "test", "epoch": 17}
{"loss": 0.19717133045196533, "acc": 0.9303880929946899, "length": 3.999357223510742, "mode": "train", "epoch": 18}
{"loss": 0.2205183207988739, "acc": 0.9206322431564331, "length": 4.0, "mode": "test", "epoch": 18}
{"loss": 0.1898844838142395, "acc": 0.9327462315559387, "length": 3.999220848083496, "mode": "train", "epoch": 19}
{"loss": 0.21878674626350403, "acc": 0.9238768815994263, "length": 4.0, "mode": "test", "epoch": 19}
{"loss": 0.18628205358982086, "acc": 0.9352487325668335, "length": 3.9988739490509033, "mode": "train", "epoch": 20}
{"loss": 0.21132776141166687, "acc": 0.9258734583854675, "length": 4.0, "mode": "test", "epoch": 20}
{"loss": 0.19348229467868805, "acc": 0.9340841174125671, "length": 3.9992644786834717, "mode": "train", "epoch": 21}
{"loss": 0.22058093547821045, "acc": 0.9214640259742737, "length": 4.0, "mode": "test", "epoch": 21}
{"loss": 0.18877653777599335, "acc": 0.9357288479804993, "length": 3.9996047019958496, "mode": "train", "epoch": 22}
{"loss": 0.21531622111797333, "acc": 0.9237934947013855, "length": 4.0, "mode": "test", "epoch": 22}
{"loss": 0.1807209998369217, "acc": 0.9376685619354248, "length": 3.999203681945801, "mode": "train", "epoch": 23}
{"loss": 0.21580971777439117, "acc": 0.9222129583358765, "length": 4.0, "mode": "test", "epoch": 23}
{"loss": 0.18062029778957367, "acc": 0.9360405206680298, "length": 3.9997589588165283, "mode": "train", "epoch": 24}
{"loss": 0.20920929312705994, "acc": 0.9261230826377869, "length": 4.0, "mode": "test", "epoch": 24}
{"loss": 0.17651665210723877, "acc": 0.9370258450508118, "length": 3.9993889331817627, "mode": "train", "epoch": 25}
{"loss": 0.20227424800395966, "acc": 0.9253743886947632, "length": 4.0, "mode": "test", "epoch": 25}
{"loss": 0.17928959429264069, "acc": 0.9377082586288452, "length": 3.99957275390625, "mode": "train", "epoch": 26}
{"loss": 0.20859020948410034, "acc": 0.925956666469574, "length": 4.0, "mode": "test", "epoch": 26}
{"loss": 0.17201711237430573, "acc": 0.9406631588935852, "length": 3.999474287033081, "mode": "train", "epoch": 27}
{"loss": 0.21882475912570953, "acc": 0.9257071018218994, "length": 4.0, "mode": "test", "epoch": 27}
{"loss": 0.17092543840408325, "acc": 0.9410971999168396, "length": 3.9994258880615234, "mode": "train", "epoch": 28}
{"loss": 0.20456930994987488, "acc": 0.9232944250106812, "length": 4.0, "mode": "test", "epoch": 28}
{"loss": 0.16619671881198883, "acc": 0.9423558712005615, "length": 3.9999256134033203, "mode": "train", "epoch": 29}
{"loss": 0.2189721018075943, "acc": 0.9198002219200134, "length": 4.0, "mode": "test", "epoch": 29}
{"loss": 0.16953636705875397, "acc": 0.9414348006248474, "length": 3.9995346069335938, "mode": "train", "epoch": 30}
{"loss": 0.21205392479896545, "acc": 0.9239600300788879, "length": 4.0, "mode": "test", "epoch": 30}
{"loss": 0.1731976568698883, "acc": 0.9390726685523987, "length": 3.999729871749878, "mode": "train", "epoch": 31}
{"loss": 0.20986559987068176, "acc": 0.9276207089424133, "length": 4.0, "mode": "test", "epoch": 31}
{"loss": 0.16540926694869995, "acc": 0.9420494437217712, "length": 3.999931812286377, "mode": "train", "epoch": 32}
{"loss": 0.20193754136562347, "acc": 0.9263728260993958, "length": 4.0, "mode": "test", "epoch": 32}
{"loss": 0.16867013275623322, "acc": 0.9391324520111084, "length": 3.999981164932251, "mode": "train", "epoch": 33}
{"loss": 0.2057945877313614, "acc": 0.9193012714385986, "length": 4.0, "mode": "test", "epoch": 33}
{"loss": 0.16881723701953888, "acc": 0.9384039044380188, "length": 3.9998209476470947, "mode": "train", "epoch": 34}
{"loss": 0.21675263345241547, "acc": 0.9305325746536255, "length": 4.0, "mode": "test", "epoch": 34}
{"loss": 0.16819694638252258, "acc": 0.9390769004821777, "length": 3.9999775886535645, "mode": "train", "epoch": 35}
{"loss": 0.1897783875465393, "acc": 0.9300333857536316, "length": 4.0, "mode": "test", "epoch": 35}
{"loss": 0.16105331480503082, "acc": 0.9438558220863342, "length": 3.9999375343322754, "mode": "train", "epoch": 36}
{"loss": 0.18758690357208252, "acc": 0.9331113696098328, "length": 4.0, "mode": "test", "epoch": 36}
{"loss": 0.16816845536231995, "acc": 0.9404106736183167, "length": 3.9999630451202393, "mode": "train", "epoch": 37}
{"loss": 0.2242172360420227, "acc": 0.9137270450592041, "length": 4.0, "mode": "test", "epoch": 37}
{"loss": 0.1764911413192749, "acc": 0.9379366636276245, "length": 3.999969482421875, "mode": "train", "epoch": 38}
{"loss": 0.2188001126050949, "acc": 0.9239599704742432, "length": 4.0, "mode": "test", "epoch": 38}
{"loss": 0.16149599850177765, "acc": 0.9428514242172241, "length": 3.999720335006714, "mode": "train", "epoch": 39}
{"loss": 0.21584849059581757, "acc": 0.9301164746284485, "length": 4.0, "mode": "test", "epoch": 39}
{"loss": 0.16798783838748932, "acc": 0.9416608810424805, "length": 3.999962091445923, "mode": "train", "epoch": 40}
{"loss": 0.2065000832080841, "acc": 0.9257903695106506, "length": 4.0, "mode": "test", "epoch": 40}
{"loss": 0.16102395951747894, "acc": 0.9420510530471802, "length": 3.9999940395355225, "mode": "train", "epoch": 41}
{"loss": 0.20454874634742737, "acc": 0.9277870655059814, "length": 4.0, "mode": "test", "epoch": 41}
{"loss": 0.16173669695854187, "acc": 0.9420782923698425, "length": 3.9999821186065674, "mode": "train", "epoch": 42}
{"loss": 0.19863326847553253, "acc": 0.9286189079284668, "length": 4.0, "mode": "test", "epoch": 42}
{"loss": 0.1577121615409851, "acc": 0.9447469115257263, "length": 3.9999961853027344, "mode": "train", "epoch": 43}
{"loss": 0.18498724699020386, "acc": 0.9364393353462219, "length": 4.0, "mode": "test", "epoch": 43}
{"loss": 0.15205125510692596, "acc": 0.9456073045730591, "length": 3.9999403953552246, "mode": "train", "epoch": 44}
{"loss": 0.2081659883260727, "acc": 0.9222129583358765, "length": 4.0, "mode": "test", "epoch": 44}
{"loss": 0.18075807392597198, "acc": 0.9335440993309021, "length": 3.9999256134033203, "mode": "train", "epoch": 45}
{"loss": 0.21892236173152924, "acc": 0.9243761301040649, "length": 4.0, "mode": "test", "epoch": 45}
{"loss": 0.16199320554733276, "acc": 0.9401730895042419, "length": 3.9994664192199707, "mode": "train", "epoch": 46}
{"loss": 0.2096700221300125, "acc": 0.9310318231582642, "length": 4.0, "mode": "test", "epoch": 46}
{"loss": 0.15810362994670868, "acc": 0.944102942943573, "length": 3.9998419284820557, "mode": "train", "epoch": 47}
{"loss": 0.2043319046497345, "acc": 0.9298669695854187, "length": 4.0, "mode": "test", "epoch": 47}
{"loss": 0.15072956681251526, "acc": 0.9454837441444397, "length": 3.999469041824341, "mode": "train", "epoch": 48}
{"loss": 0.2012471705675125, "acc": 0.9319469332695007, "length": 4.0, "mode": "test", "epoch": 48}
{"loss": 0.15411436557769775, "acc": 0.9456638693809509, "length": 3.999983787536621, "mode": "train", "epoch": 49}
{"loss": 0.22500313818454742, "acc": 0.9202994704246521, "length": 4.0, "mode": "test", "epoch": 49}
{"loss": 0.15883862972259521, "acc": 0.941739022731781, "length": 3.9997525215148926, "mode": "train", "epoch": 50}
{"loss": 0.20142687857151031, "acc": 0.9321130514144897, "length": 4.0, "mode": "test", "epoch": 50}
{"loss": 0.15288041532039642, "acc": 0.9452738165855408, "length": 3.9999279975891113, "mode": "train", "epoch": 51}
{"loss": 0.2029431164264679, "acc": 0.9288685321807861, "length": 4.0, "mode": "test", "epoch": 51}
{"loss": 0.1675819456577301, "acc": 0.9403525590896606, "length": 3.999905586242676, "mode": "train", "epoch": 52}
{"loss": 0.2099575698375702, "acc": 0.9287024140357971, "length": 4.0, "mode": "test", "epoch": 52}
{"loss": 0.15470027923583984, "acc": 0.9443199038505554, "length": 3.9993982315063477, "mode": "train", "epoch": 53}
{"loss": 0.19157563149929047, "acc": 0.93186354637146, "length": 4.0, "mode": "test", "epoch": 53}
{"loss": 0.14544963836669922, "acc": 0.9483010768890381, "length": 3.9995718002319336, "mode": "train", "epoch": 54}
{"loss": 0.19234976172447205, "acc": 0.9331114888191223, "length": 4.0, "mode": "test", "epoch": 54}
{"loss": 0.1449383795261383, "acc": 0.9477208852767944, "length": 3.9999959468841553, "mode": "train", "epoch": 55}
{"loss": 0.18023528158664703, "acc": 0.9382694959640503, "length": 4.0, "mode": "test", "epoch": 55}
{"loss": 0.14467214047908783, "acc": 0.9480822682380676, "length": 3.9999961853027344, "mode": "train", "epoch": 56}
{"loss": 0.1969551295042038, "acc": 0.9330283999443054, "length": 4.0, "mode": "test", "epoch": 56}
{"loss": 0.1412307620048523, "acc": 0.9502778053283691, "length": 3.9999966621398926, "mode": "train", "epoch": 57}
{"loss": 0.1845620572566986, "acc": 0.9385192394256592, "length": 4.0, "mode": "test", "epoch": 57}
{"loss": 0.14819838106632233, "acc": 0.9457475543022156, "length": 3.999997615814209, "mode": "train", "epoch": 58}
{"loss": 0.2010752111673355, "acc": 0.9227122068405151, "length": 4.0, "mode": "test", "epoch": 58}
{"loss": 0.1464855968952179, "acc": 0.9463585019111633, "length": 3.999962568283081, "mode": "train", "epoch": 59}
{"loss": 0.23309044539928436, "acc": 0.9203826189041138, "length": 4.0, "mode": "test", "epoch": 59}
{"loss": 0.1512693613767624, "acc": 0.9463587403297424, "length": 3.9999895095825195, "mode": "train", "epoch": 60}
{"loss": 0.20990364253520966, "acc": 0.9287853837013245, "length": 4.0, "mode": "test", "epoch": 60}
{"loss": 0.14270171523094177, "acc": 0.9482489824295044, "length": 3.9999969005584717, "mode": "train", "epoch": 61}
{"loss": 0.1902795135974884, "acc": 0.9334442615509033, "length": 4.0, "mode": "test", "epoch": 61}
{"loss": 0.14030486345291138, "acc": 0.9506374597549438, "length": 3.9999077320098877, "mode": "train", "epoch": 62}
{"loss": 0.1805344969034195, "acc": 0.9361897110939026, "length": 4.0, "mode": "test", "epoch": 62}
{"loss": 0.1406070590019226, "acc": 0.9499443769454956, "length": 3.9999992847442627, "mode": "train", "epoch": 63}
{"loss": 0.18633466958999634, "acc": 0.9297003746032715, "length": 4.0, "mode": "test", "epoch": 63}
{"loss": 0.15715375542640686, "acc": 0.9425234198570251, "length": 3.9999935626983643, "mode": "train", "epoch": 64}
{"loss": 0.19898517429828644, "acc": 0.9222961664199829, "length": 4.0, "mode": "test", "epoch": 64}
{"loss": 0.1418701857328415, "acc": 0.9468590617179871, "length": 3.999966859817505, "mode": "train", "epoch": 65}
{"loss": 0.21588629484176636, "acc": 0.9282028675079346, "length": 4.0, "mode": "test", "epoch": 65}
{"loss": 0.1492878496646881, "acc": 0.9455251097679138, "length": 3.9999921321868896, "mode": "train", "epoch": 66}
{"loss": 0.1857195347547531, "acc": 0.9331115484237671, "length": 4.0, "mode": "test", "epoch": 66}
{"loss": 0.14490032196044922, "acc": 0.9475658535957336, "length": 3.9996562004089355, "mode": "train", "epoch": 67}
{"loss": 0.18656665086746216, "acc": 0.9301996231079102, "length": 4.0, "mode": "test", "epoch": 67}
{"loss": 0.14041081070899963, "acc": 0.949499785900116, "length": 3.9999983310699463, "mode": "train", "epoch": 68}
{"loss": 0.17526857554912567, "acc": 0.9346088171005249, "length": 4.0, "mode": "test", "epoch": 68}
{"loss": 0.13661907613277435, "acc": 0.9488601684570312, "length": 3.999986171722412, "mode": "train", "epoch": 69}
{"loss": 0.19676898419857025, "acc": 0.9307822585105896, "length": 4.0, "mode": "test", "epoch": 69}
{"loss": 0.1433507651090622, "acc": 0.9508339166641235, "length": 4.0, "mode": "train", "epoch": 70}
{"loss": 0.18845577538013458, "acc": 0.933527410030365, "length": 4.0, "mode": "test", "epoch": 70}
{"loss": 0.1490175426006317, "acc": 0.947053849697113, "length": 3.9999985694885254, "mode": "train", "epoch": 71}
{"loss": 0.1874827742576599, "acc": 0.9386023879051208, "length": 4.0, "mode": "test", "epoch": 71}
{"loss": 0.13539113104343414, "acc": 0.9519733786582947, "length": 4.0, "mode": "train", "epoch": 72}
{"loss": 0.1748499572277069, "acc": 0.9379366040229797, "length": 4.0, "mode": "test", "epoch": 72}
{"loss": 0.13391783833503723, "acc": 0.9533073306083679, "length": 3.9999983310699463, "mode": "train", "epoch": 73}
{"loss": 0.1842397004365921, "acc": 0.9349415898323059, "length": 4.0, "mode": "test", "epoch": 73}
{"loss": 0.1427336484193802, "acc": 0.9479984641075134, "length": 3.99997878074646, "mode": "train", "epoch": 74}
{"loss": 0.2171231061220169, "acc": 0.9207154512405396, "length": 4.0, "mode": "test", "epoch": 74}
{"loss": 0.15115366876125336, "acc": 0.9458032846450806, "length": 3.9999992847442627, "mode": "train", "epoch": 75}
{"loss": 0.19120080769062042, "acc": 0.9274542927742004, "length": 4.0, "mode": "test", "epoch": 75}
{"loss": 0.1380872279405594, "acc": 0.9498610496520996, "length": 4.0, "mode": "train", "epoch": 76}
{"loss": 0.17853856086730957, "acc": 0.9364393353462219, "length": 4.0, "mode": "test", "epoch": 76}
{"loss": 0.13827887177467346, "acc": 0.9476916790008545, "length": 3.9999613761901855, "mode": "train", "epoch": 77}
{"loss": 0.18893016874790192, "acc": 0.9360232949256897, "length": 4.0, "mode": "test", "epoch": 77}
{"loss": 0.1409742832183838, "acc": 0.9486657381057739, "length": 3.999990701675415, "mode": "train", "epoch": 78}
{"loss": 0.1863040179014206, "acc": 0.9320299625396729, "length": 4.0, "mode": "test", "epoch": 78}
{"loss": 0.1361478716135025, "acc": 0.9508615732192993, "length": 3.9999992847442627, "mode": "train", "epoch": 79}
{"loss": 0.1764267086982727, "acc": 0.9371049404144287, "length": 4.0, "mode": "test", "epoch": 79}
{"loss": 0.13313734531402588, "acc": 0.9496108293533325, "length": 4.0, "mode": "train", "epoch": 80}
{"loss": 0.17406615614891052, "acc": 0.9374374151229858, "length": 4.0, "mode": "test", "epoch": 80}
{"loss": 0.14698219299316406, "acc": 0.9467759728431702, "length": 4.0, "mode": "train", "epoch": 81}
{"loss": 0.19159916043281555, "acc": 0.9287853837013245, "length": 4.0, "mode": "test", "epoch": 81}
{"loss": 0.14227670431137085, "acc": 0.9464147090911865, "length": 3.999999523162842, "mode": "train", "epoch": 82}
{"loss": 0.18204949796199799, "acc": 0.9371881484985352, "length": 4.0, "mode": "test", "epoch": 82}
{"loss": 0.1415809988975525, "acc": 0.9478042721748352, "length": 3.9999983310699463, "mode": "train", "epoch": 83}
{"loss": 0.24821095168590546, "acc": 0.8982528448104858, "length": 4.0, "mode": "test", "epoch": 83}
{"loss": 0.1773475706577301, "acc": 0.9257359504699707, "length": 3.999986410140991, "mode": "train", "epoch": 84}
{"loss": 0.19653508067131042, "acc": 0.921464204788208, "length": 4.0, "mode": "test", "epoch": 84}
{"loss": 0.15771664679050446, "acc": 0.9344059228897095, "length": 3.9999308586120605, "mode": "train", "epoch": 85}
{"loss": 0.19790035486221313, "acc": 0.9212146997451782, "length": 4.0, "mode": "test", "epoch": 85}
{"loss": 0.15567143261432648, "acc": 0.9353528618812561, "length": 3.9999992847442627, "mode": "train", "epoch": 86}
{"loss": 0.20637856423854828, "acc": 0.9188851714134216, "length": 4.0, "mode": "test", "epoch": 86}
{"loss": 0.15580350160598755, "acc": 0.9361028671264648, "length": 3.9999642372131348, "mode": "train", "epoch": 87}
{"loss": 0.2017005831003189, "acc": 0.9228785037994385, "length": 4.0, "mode": "test", "epoch": 87}
{"loss": 0.15505586564540863, "acc": 0.9347136616706848, "length": 3.9999988079071045, "mode": "train", "epoch": 88}
{"loss": 0.2093251645565033, "acc": 0.9198834896087646, "length": 4.0, "mode": "test", "epoch": 88}
{"loss": 0.14210452139377594, "acc": 0.9446914792060852, "length": 4.0, "mode": "train", "epoch": 89}
{"loss": 0.19359125196933746, "acc": 0.9338601231575012, "length": 4.0, "mode": "test", "epoch": 89}
{"loss": 0.13888001441955566, "acc": 0.9498610496520996, "length": 4.0, "mode": "train", "epoch": 90}
{"loss": 0.22758127748966217, "acc": 0.9290348291397095, "length": 4.0, "mode": "test", "epoch": 90}
{"loss": 0.16074329614639282, "acc": 0.94129878282547, "length": 3.99995756149292, "mode": "train", "epoch": 91}
{"loss": 0.21468235552310944, "acc": 0.9244592189788818, "length": 4.0, "mode": "test", "epoch": 91}
{"loss": 0.14829932153224945, "acc": 0.9480543732643127, "length": 3.999998092651367, "mode": "train", "epoch": 92}
{"loss": 0.21022817492485046, "acc": 0.9272047877311707, "length": 4.0, "mode": "test", "epoch": 92}
{"loss": 0.1377594918012619, "acc": 0.9485545754432678, "length": 3.999995708465576, "mode": "train", "epoch": 93}
{"loss": 0.18122315406799316, "acc": 0.9371881484985352, "length": 4.0, "mode": "test", "epoch": 93}
{"loss": 0.13874366879463196, "acc": 0.9481656551361084, "length": 4.0, "mode": "train", "epoch": 94}
{"loss": 0.18062825500965118, "acc": 0.9340264797210693, "length": 4.0, "mode": "test", "epoch": 94}
{"loss": 0.13567720353603363, "acc": 0.9493597745895386, "length": 3.999976634979248, "mode": "train", "epoch": 95}
{"loss": 0.17704375088214874, "acc": 0.9371881484985352, "length": 4.0, "mode": "test", "epoch": 95}
{"loss": 0.13757175207138062, "acc": 0.9476373791694641, "length": 3.999995470046997, "mode": "train", "epoch": 96}
{"loss": 0.17661646008491516, "acc": 0.9371882081031799, "length": 4.0, "mode": "test", "epoch": 96}
{"loss": 0.12949238717556, "acc": 0.9516663551330566, "length": 3.9999802112579346, "mode": "train", "epoch": 97}
{"loss": 0.18045882880687714, "acc": 0.9354408383369446, "length": 4.0, "mode": "test", "epoch": 97}
{"loss": 0.1318202167749405, "acc": 0.9524456858634949, "length": 4.0, "mode": "train", "epoch": 98}
{"loss": 0.19084951281547546, "acc": 0.9380200505256653, "length": 4.0, "mode": "test", "epoch": 98}
{"loss": 0.13558198511600494, "acc": 0.9511670470237732, "length": 3.9999983310699463, "mode": "train", "epoch": 99}
{"loss": 0.18589958548545837, "acc": 0.9374376535415649, "length": 4.0, "mode": "test", "epoch": 99}
{"loss": 0.14605507254600525, "acc": 0.9448025226593018, "length": 3.999997615814209, "mode": "train", "epoch": 100}
{"loss": 0.18228082358837128, "acc": 0.9358569979667664, "length": 4.0, "mode": "test", "epoch": 100}
test accuracy: 0.8317497968673706
